Apps/FCM/src/screens/InternalSettings.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from 'react';
+import { View, Text, ScrollView } from 'react-native';
+import { TextField } from '../components/TextField';
+import { FilledButton } from '../components/Button';
+import {styles as settingsStyles} from '../styles/stylesheet';
+
+const InternalSettings: React.FC = () => {
+    const [apiHost, setApiHost] = useState('');
+    const [cdnHost, setCdnHost] = useState('');
+    const [flushAt, setFlushAt] = useState('');
+    const [flushInterval, setFlushInterval] = useState('');
+
+    const handleSave = () => {
+
+    };
+    
+
+    return (
+        <View style={settingsStyles.container}>
+            <ScrollView contentContainerStyle={settingsStyles.content}>
+                <View style={settingsStyles.section}>
+                    <TextField
+                        style={settingsStyles.textInputContainer}
+                        label="CDN Host"
+                        value={cdnHost}
+                        contentDesc="CDN Host Input"
+                        placeholder='CDN Host'
+                        onChangeText={text => setCdnHost(text)}
+                        // textInputRef={siteIdRef}
```

Comment:
Do you plan to clean up this and similar comments below in later PRs with save action implementation? Since the UI is mostly completely, I see a chance of this being missed. Can you please remove them now or leave comments so we don't forget to address this later?

Reply:
Fair suggestion. I have removed this for now and can add later as required.